### DISCUSSION TITLE
return a variable I can execute off of

### DISCUSSION THREAD

**Discussion by IotaSpencer** [sqlalchemy/sqlalchemy#8439] [Usage Questions]
How would I go about getting a variable that I can just do `await var.execute(select().where(...))`

I have 
##### db.py
```python
async def db_engine():
    engine = create_async_engine(
        f'{dbcfg.proto}://{dbcfg.user}:{dbcfg.password}@{dbcfg.host}/{dbcfg.database}?charset=utf8mb4')
    return engine
```

If there is a discussion or piece of documentation that shows how, then please give links to it

**Comment by CaselIT** [comment]
Hi

you are probably looking for this https://docs.sqlalchemy.org/en/14/tutorial/index.html and the async docs https://docs.sqlalchemy.org/en/14/orm/extensions/asyncio.html

**Reply by zzzeek** [reply]
try the examples  https://docs.sqlalchemy.org/en/14/_modules/examples/asyncio/basic.html
